tests/test-definition.yaml

```diff
@@ -54,3 +54,8 @@ tests:
       - use-client-tls
       - use-client-auth-tls
       - use-internal-tls
+  - name: delete-rolegroup
+    dimensions:
+      - kafka
+      - zookeeper-latest
+      - use-internal-tls
```

Comment:
Do we really need internal tls here? Cleaning up the resources is independent of that right?

Reply:
Removed.